ee/fleetd-chrome/src/tables/os_version.ts

```diff
@@ -100,7 +100,7 @@ export default class TableOSVersion extends Table {
       // we just hardcode to "chrome" so that Fleet always sees this Chrome extension as a Chrome
       // device even when we are doing local dev on a non-ChromeOS machine.
       const platformInfo = await chrome.runtime.getPlatformInfo();
-      arch = platformInfo;
+      arch = platformInfo.arch;
```

Comment:
Will look into this

Reply:
https://developer.chrome.com/docs/extensions/reference/api/runtime#type-PlatformInfo

Reply:
I'm trying to test to see what returns from the API calls but my Chromebook is not playing nice. According to the API dock, `arch` is one of the properties in the object returned from  `getPlatformInfo()`

Reply:
Alright, got my extension up and running today. I'm going to stand by my change `arch = platformInfo.arch` since I was able to confirm the API docs that platformInfo returns an object with arch as a property.

<img width="817" alt="Screenshot 2024-02-23 at 9 51 51 AM" src="https://github.com/fleetdm/fleet/assets/71795832/c2592bef-2c95-4460-80cf-18fc68c32500">